nbproject/project.properties

```diff
@@ -94,7 +94,7 @@ javadoc.encoding=${source.encoding}
 build.test.classes.dir=${build.dir}/test/classes
 jar.index=${jnlp.enabled}
 file.reference.jrcs.jar=lib/jrcs.jar
-javac.compilerargs=-Xlint\:unchecked
+javac.compilerargs=-Xlint:-fallthrough
```

Comment:
please keep both options (  http://wiki.netbeans.org/Java_Hints#Standard_Javac_warnings )

Reply:
Hmm.... I didn't realize that this got pushed up too. I thought it was just my local compilation. So are you asking to put both unchecked and -fallthrough together as part of the compilation. Or should I just revert to the unchecked option?

Reply:
Is there a way to assign compile options to specific files like some C/C++ compilers do? For example, I would really like the -fallthrough option only be applied to those files that are generated by JFlex.

Reply:
hmm, maybe it can be done, if you want we can move this to a subsequent push request to expedite this one